### DISCUSSION TITLE
No more hacking and infecting things

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13515] [Other Feature Feedback, Questions, & Ideas]
My Recommended For You keeps showing hacks and such: pls stop

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
